### PR TITLE
clean up e2e test

### PR DESCRIPTION
**Pull request by velimd**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/steps/IdamUserManager.ts [hunk 1]
```diff
@@ -50,11 +50,15 @@ export class IdamUserManager {
     }
   }
 
+  async clearAndKeepOnlyOriginalUser(): Promise<void> {
+    this.users = new Set([...this.users][0]);
```
Comment (linusnorton) [inline src/test/steps/IdamUserManager.ts:54]: new Set(this.users[0])?

Comment (velimd) [reply src/test/steps/IdamUserManager.ts:54]: creates a new set with the first element from the previous users.

Comment (velimd) [reply src/test/steps/IdamUserManager.ts:54]: i guess i can use the method below getUsername(0) to also get it.

Comment (linusnorton) [reply src/test/steps/IdamUserManager.ts:54]: no it's okay, I didn't realise it was a set

**Review by linusnorton — APPROVED** [review]

**Pull request merged**